lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -763,7 +763,6 @@ function dosomething_campaign_is_hot_module($vars) {
     $today = dosomething_campaign_set_est_timezone(new DateTime());
     $start_date = dosomething_campaign_set_est_timezone(new DateTime($vars['field_high_season'][0]['value']), TRUE);
     $end_date = dosomething_campaign_set_est_timezone(new DateTime($vars['field_high_season'][0]['value2']), TRUE, TRUE);
-    $end_date->setTime(23, 59, 59);
```

Comment:
This is already done in the function it calls (`dosomething_campaign_set_est_timezone`)